Makefile

```diff
@@ -10,8 +10,9 @@ version:
 run: version
 	cargo run --bin cli
 
-install: version
-	cargo install --locked --path crates/ursa --force
+install: version build
```

Comment:
Since we're just building without --locked and manually copying, couldn't we just use install without --locked instead, since that works fine?

Reply:
no `cargo install` just doesn't work, with or without `--locked` 😭

Reply:
oh, i think you're right :cry: